indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -218,3 +221,15 @@ def _process(self):
         if not self.booking.event.can_access(session.user):
             return jsonify(can_access=False)
         return jsonify(reservation_event_data_schema.dump(self.booking.event).data)
+
+
+class RHMyUpcomingBookings(RHRoomBookingBase):
+    def _process(self):
+        q = ReservationOccurrence.query.filter(
+            ReservationOccurrence.start_dt > now_utc(),
```

Comment:
aren't those datetimes naive? ==> now instead of now_utc?

Reply:
Good point. I guess it should rather be `pytz.timezone(config.DEFAULT_TIMEZONE).localize(now_utc())` in that case.

Reply:
We have a `utc_to_server()` util that does it

Reply:
Ah cool.